hbase-client/src/main/java/org/apache/hadoop/hbase/client/CheckAndMutate.java

```diff
@@ -60,7 +53,7 @@
  */
 @InterfaceAudience.Public
 @InterfaceStability.Evolving
-public final class CheckAndMutate extends Mutation {
+public final class CheckAndMutate implements Row {
```

Comment:
Why this change?

Reply:
OK, I'm fine with it, but since this is IA.Public, we need to make sure that we haven't release this class out yet? There will be a 2.4.0 release soon.

Reply:
We haven't released the CheckAndMutate class yet. It's introduced in 2.4.0. We need to commit this change before releasing 2.4.0.